README.md

```diff
@@ -67,6 +67,7 @@ module "private_subnets" {
 | enable_classiclink | A boolean flag to enable/disable ClassicLink for the VPC | bool | `false` | no |
 | enable_classiclink_dns_support | A boolean flag to enable/disable ClassicLink DNS Support for the VPC | bool | `false` | no |
 | enable_default_security_group_with_custom_rules | A boolean flag to enable/disable custom and restricive inbound/outbound rules for the default VPC's SG | bool | `true` | no |
+| enable_internet_gateway | A boolean flag to enable/disable the creation of an internet gateway | bool | `true` | no |
```

Comment:
Le podemos poner `internet_gateway_enabled` para que quede consistente con https://github.com/ceibo-it/terraform-aws-subnets/blob/master/variables.tf#L153 ?

Reply:
Jeje,  y yo lo hice consistente con:

```variable "nat_gateway_enabled" {
  description = "Flag to enable/disable NAT Gateways creation in public subnets"
  default     = "true"
}```

Reply:
Ahh soy un dislexico del orto

Reply:
Lo copy pastie de `enable_default_security_group_with_custom_rules`